scargo/jinja/docker/Dockerfile.j2

```diff
@@ -60,7 +60,10 @@ RUN printf "\n\nADDING USER $USER_NAME TO SUDOERS - DEV ENV ONLY!\n\n" >&2 && \
     echo "$USER_NAME:$USER_PASSWORD" | chpasswd && \
     echo "$USER_NAME ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
 
-RUN pip3 install scargo=={{ scargo_version }}
+{% if "whl" in scargo_package_version %}
```

Comment:
Maybe something like `{% if scargo_package_version.endswith(".whl") %}`? It's more straightforward and precise.